webview/src/shared/components/sectionContainer/styles.module.scss

```diff
@@ -52,6 +52,18 @@ $gap: 20px;
 .infoIcon {
   fill: $accent-color;
   margin-left: 6px;
+
+  .infoTooltip & {
+    margin-left: 0;
```

Comment:
Is there an `infoIcon` anywhere else other than inside `infoTooltip`? If I missed it, better add another class to this one instead. If there isn't, we should remove that nesting.

Reply:
Yes, it's used inside the `infoTooltip` and inside the header. I'll add another class.